platforms/azure/modules/machine/machine.tf

```diff
@@ -70,7 +70,7 @@ data "template_file" "class" {
     offer = "${local.offer}"
     sku = "${local.sku}"
     version = "${local.version}"
-
+    public_key = ""
```

Comment:
Is really fine having an empty value as a default?

Reply:
I logged into bastion/worker/master nodes without issues. So seems like this is irrelevant.

Reply:
This one should be removed from the right place instead of assigning an empty string or deserves a comment about why this cant be removed and instead assigned an empty value.